learners/setup.md

```diff
@@ -63,20 +109,35 @@ source dl_workshop/bin/activate
 ### On Windows
 
 ```shell
-dl_workshop\Scripts\activate
+venv\Scripts\activate
 ```
 
+If you're using Git Bash on Windows, you need to add the source command first.
+
+```shell
+source venv/Scripts/activate
+```
 :::
 
-Remember that you need to activate your environment every time you restart your terminal!
+**Note**: Remember that you need to activate your environment every time you restart your terminal, and before you launch Jupyter Lab!
+
+3. After activating the enviornment, upgrade pip. This is a good practice to follow when you first initialize your virtual environment (beforing installing additional packages). [Pip](https://pip.pypa.io/en/stable/) is the package management system built into Python.Pip should be available in your system once you installed Python successfully.
 
-3. Install the required packages:
+```shell
+ # remember: use python3 or py instead if it points to 3.11.9
+python -m pip install --upgrade pip
+```
+
+4. Install the required packages:
+
+Follow the OS-specific instructions below. NOte that It may take 10-20 minutes to install everything.
```

Comment:
Typo.
```suggestion
Follow the OS-specific instructions below. Nte that It may take 10-20 minutes to install everything.
```
Also, 10-20 minutes seems a bit pessimistic.

Reply:
```suggestion
Follow the OS-specific instructions below. Note that It may take 10-20 minutes to install everything.
```